fern/core/addons.js

```diff
@@ -1,6 +1,7 @@
 const stream = require("stream");
 const path = require("path");
 const fs = require("fs");
+const md5 = require("md5");
```

Comment:
In principle I think we should be able to use the standard module https://nodejs.org/api/crypto.html for that?

Reply:
For example:
```js
const crypto = require('crypto');

const getAddonUrlHash = url => crypto.createHash('md5').update(url).digest("hex");
```

Reply:
yeah, good point